src/disco/css/Addon.scss

```diff
@@ -8,23 +8,33 @@ $secondary-text-color: #6a6a6a;
   overflow: hidden;
 
   .theme-image {
+    display: block;
     width: 100%;
   }
 
+  .install-button {
+    margin: 0 20px 0 30px;
+  }
+
   .content {
     align-items: center;
     display: flex;
     flex-direction: row;
     line-height: 1.5;
-    padding: 20px;
 
     .logo {
-      height: 64px;
-      width: 64px;
+      background: #fcfcfc;
```

Comment:
This background isn't covering the full height of the block. I think the parent needs to `align-items: stretch` and then the logo and button wrappers will `display: flex; align-items: center`.

```scss
.content {
  align-items: stretch;

  // Maybe even `.content > *` instead.
  .install-button,
  .logo {
    display: flex;
    align-items: center;
  }
}